src/mono/mono/mini/mini-arm64.h

```diff
@@ -268,8 +276,8 @@ struct CallInfo {
 };
 
 typedef struct {
-	/* General registers + ARMREG_R8 for indirect returns */
-	host_mgreg_t gregs [PARAM_REGS + 1];
+	/* General registers + context registers + ARMREG_R8 for indirect returns */
```

Comment:
This comment is slightly misleading. Maybe it is better: 
`/* General registers + ARMREG_R8 for indirect returns + context registers  */`